### PR TITLE
[EASI-3965] New financial guarantee question for ISP

### PR DESCRIPTION
**Pull request by StevenWadeOddball**
# EASI-3965, EASI-3964

## Changes and Description

- Added new questions on the backend
   - Update unit tests
- Added new question with tooltip 
- Added the new question to the task list form and read only

### BE
1. Bring up the Application
2. Use Postman to Create a Model plan, and update Plan Payments and providers. (the postman query was updated to provide an answer for each question, so you only need to click send).
3. Assert that the new types persist and is returnable from Postman

### FE

1. Bring up the app
2. Go to Page 4 of Participants and Providers
3. Fill out the new question -- check out the tooltip
4. Verify data is saved
5. Check out the read only and see the data is there
6. Check out the `IPC` filter view 
7. Make sure all things match figma

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

**Comment by OddTomBrooks** [comment]
As discussed on Friday, as this is part of a full-stack pass in progress I am approving the backend work via this comment. I have no suggestions for the code itself, the unit tests pass, and the postman query functions as expected.

**Comment by StevenWadeOddball** [comment]
Converting to Draft til the FE is implemented

File: src/components/AddNote/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -3,7 +3,7 @@
 exports[`The AddNote component > matches snapshot 1`] = `
 <DocumentFragment>
   <div
-    class="margin-top-4 margin-bottom-8"
+    class="margin-top-3 margin-bottom-6"
```
Comment (garyjzhao) [inline src/components/AddNote/__snapshots__/index.test.tsx.snap:6]: image

File: src/components/AddNote/index.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ const AddNote = ({ field: fieldName, id, className }: AddNoteType) => {
   const { setValue } = helpers;
 
   return (
-    <div className={classNames('margin-top-4 margin-bottom-8', className)}>
+    <div className={classNames('margin-top-3 margin-bottom-6', className)}>
```
Comment (garyjzhao) [inline src/components/AddNote/index.tsx:43]: image

Comment (garyjzhao) [reply src/components/AddNote/index.tsx:43]: This change to match resulted to a bunch of snapshot updates. Just ICYWW

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]
The readonly render does not match Figma.  In Figma, UX has combined both questions into one, absolving the need for a `parentRelation`/`childRelation` field.  Instead the `readonly` label should be used to represent both questions

https://www.figma.com/file/6tkIhfNfcsKEAkqXo1qvXH/MINT---Read-Only-View?type=design&node-id=4247-64648&mode=design&t=YHREGFRhAmn29gvJ-0

The parent/child relations controls these alerts that allow you to toggle hidden questions.  This alert should not be shown as both questions are combined in readonly
Screenshot 2024-04-01 at 11 20 35 AM

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]
These new fields need to be included in the csv export

**Review by patrickseguraoddball — APPROVED** [review]
LG!

**Pull request merged**